src/main/java/org/helpberkeley/memberdata/WorkflowExporter.java

```diff
@@ -166,4 +177,8 @@ private String updatedWorkflowToString() {
     public void changeMemberLimit(int limit) {
         member_limit = limit;
     }
+
+    public void changeIncomingHeader(String incomingHeader) {
+        this.incomingHeader = incomingHeader;
+    }
```

Comment:
I don't have problems with adding whitebox testing support like this, but I don't think that this is testing the case we are looking for.  More details on this in my comment in MainTest